### PR TITLE
Lineage integrations documentation

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/501

This is an Astro docs version of our embedded Datakin docs: https://astro-dev-astronomer.datakin.com/onboarding

The base content hasn't changed much. Primary changes are:

- Updated overview section to include less introductory information about lineage
- Added some shoutouts to the embedded docs
- Reworded steps on how to configure the OpenLineage API key
- Updated links to be internal to Astro documentation. removed some remnants from DK

### CODE REVIEW

File: astro/lineage-integrations.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+Each Organization has a unique OpenLineage API key that is required to send lineage data from external systems to Astro. To find your Organization's key:
+
+1. In the Cloud UI, open the **Lineage** tab.
+2. In the left-hand lineage menu, click **Integrations**:
+
+    ![Location of the "Integrations" button in the Lineage tab of the Cloud UI](/img/docs/lineage-integrations.png)
+
+3. In **Getting Started**, copy the value in **Lineage API Key**.
+
+This key is required in each of the following integration guides.
```
Comment (jwitz) [inline astro/lineage-integrations.md:39]:
```suggestion
For more information about how to configure this API key in external systems, read the following integration guides. 
```

**Review by rossturk — APPROVED** [review]
Lgtm!

File: astro/lineage-integrations.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
+---
+sidebar_label: 'Set Up Lineage'
+title: "Enable Data Lineage for External Systems"
+id: lineage-integrations
```
Comment (jwitz) [inline astro/lineage-integrations.md:4]:
```suggestion
sidebar_label: 'Set Up Data Lineage'
title: "Enable Data Lineage for External Systems"
id: set-up-data-lineage
```

**Review by TohnJhomas — APPROVED** [review]
LGTM

File: astro/lineage-integrations.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+   ```
+
+2. [Push your changes](https://docs.astronomer.io/cloud/deploy-code) to your Deployment.
+
+3. In the Cloud UI, [set the following environment variables](https://www.astronomer.io/docs/cloud/stable/deploy/environment-variables) in your Deployment:
+
+    ```
+    AIRFLOW__LINEAGE__BACKEND=openlineage.lineage_backend.OpenLineageBackend
+    OPENLINEAGE_NAMESPACE=<your-deployment-namespace>
+    OPENLINEAGE_URL=https://api.<your-astro-base-domain>
```
Comment (conorbev) [inline astro/lineage-integrations.md:73]: remove `api.` part so this becomes: `OPENLINEAGE_URL=https://<your-astro-base-domain>`

File: astro/lineage-integrations.md [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+
+    /databricks/python/bin/pip install openlineage-spark
+    ```
+
+2. In the cluster configuration page for your Databricks cluster, specify the following [Spark configuration](https://docs.databricks.com/clusters/configure.html#spark-configuration):
+
+   ```bash
+   spark.driver.extraJavaOptions -Djava.security.properties=
+   spark.executor.extraJavaOptions -Djava.security.properties=
+   spark.openlineage.url https://api.<your-astro-base-domain>
```
Comment (conorbev) [inline astro/lineage-integrations.md:114]: remove `api.` part so this becomes: `spark.openlineage.url https://<your-astro-base-domain>`

File: astro/lineage-integrations.md [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+1. On your local machine, run the following command to install the [`openlineage-dbt`](https://pypi.org/project/openlineage-dbt) library:
+
+   ```sh
+   $ pip install openlineage-dbt
+   ```
+
+2. Configure the following environment variables in your shell:
+
+   ```bash
+   OPENLINEAGE_URL=https://api.<your-astro-base-domain>
```
Comment (conorbev) [inline astro/lineage-integrations.md:151]: remove `api.` part so this becomes: `OPENLINEAGE_URL=https://<your-astro-base-domain>`

File: astro/lineage-integrations.md [hunk 6]
```diff
@@ -0,0 +1,258 @@
...
+    ```yml
+    validation_operators:
+      action_list_operator:
+        class_name: ActionListValidationOperator
+        action_list:
+          - name: openlineage
+            action:
+              class_name: OpenLineageValidationAction
+              module_name: openlineage.common.provider.great_expectations
+              openlineage_host: https://api.<your-astro-base-domain>
```
Comment (conorbev) [inline astro/lineage-integrations.md:200]: remove `api.` part so this becomes: `openlineage_host: https://<your-astro-base-domain>`

File: astro/lineage-integrations.md [hunk 7]
```diff
@@ -0,0 +1,258 @@
...
+
+#### Setup
+
+In your Spark application, set the following properties to configure your lineage endpoint, install the [`openlineage-spark`](https://search.maven.org/artifact/io.openlineage/openlineage-spark) library, and configure an _OpenLineageSparkListener_:
+
+   ```python
+   SparkSession.builder \
+     .config('spark.jars.packages', 'io.openlineage:openlineage-spark:0.2.+')
+     .config('spark.extraListeners', 'io.openlineage.spark.agent.OpenLineageSparkListener')
+     .config('spark.openlineage.host', 'https://api.<your-astro-base-domain>')
```
Comment (conorbev) [inline astro/lineage-integrations.md:247]: remove `api.` part so this becomes: `.config('spark.openlineage.host', 'https://<your-astro-base-domain>')`

**Review by conorbev — CHANGES_REQUESTED** [review]
The URL for ingesting data should no longer be prefixed with `api.`

I left comments on the areas in this doc that need updating as a result.

File: astro/lineage-integrations.md [hunk 8]
```diff
@@ -0,0 +1,259 @@
...
+
+4. In your dbt project, run the [OpenLineage](https://openlineage.io/integration/dbt/) wrapper script using the `dbt run` [command](https://docs.getdbt.com/reference/commands/run):
+
+   ```bash
+   $ dbt-ol run
+   ```
+
+#### Verify Setup
+
+To confirm that your setup was successful, run a `dbt` model in your project. After you run this model, open the **Lineage** tab in the Cloud UI and go to the **Explore** page. If the setup was successful, you should see the run that you triggered in the **Most Recent Runs** table.
```
Comment (jwitz) [inline astro/lineage-integrations.md:172]:
```suggestion
To confirm that your setup was successful, run a dbt model in your project. After you run this model, open the **Lineage** tab in the Cloud UI and go to the **Explore** page. If the setup was successful, you should see the run that you triggered in the **Most Recent Runs** table.
```

File: sidebarsAstro.js [hunk 9]
```diff
@@ -73,6 +73,7 @@ module.exports = {
       items: [
         'install-aws',
         'manage-workspaces',
+        'lineage-integrations',
```
Comment (jwitz) [inline sidebarsAstro.js:76]:
```suggestion
        'set-up-data-lineage',
```

**Review by paolaperaza — APPROVED** [review]

**Review by conorbev — APPROVED** [review]
lgtm

**Pull request merged**